### DISCUSSION TITLE
[.NET7 setup] Bug in documention

### DISCUSSION THREAD

**Discussion by DierkDroth** [unoplatform/uno#10448] [General]
The parameter `-framework net7.0` as documented [here ](https://platform.uno/docs/articles/get-started-dotnet-new.html?tabs=windows)appears to be incorrect (see screenshot below).

It appears `-f net7.0` would be working though. However, it still says 'NET 6' (screenshot).

Is this now .NET6 or .NET 7?

<img width="867" alt="image" src="https://user-images.githubusercontent.com/26179242/201982582-3a1f983f-d5df-48f8-9c8e-03b27b40b180.png">

**Comment by jeromelaban** [comment]
Thanks for the report, it will be fixed in https://github.com/unoplatform/uno/pull/10476
